.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We usually do not provide this file and the Puppet extension has moved to the official Puppet namespace.